Realm/Realm/Schema/RealmSchema.cs

```diff
@@ -212,12 +210,18 @@ public RealmSchema Build()
             {
                 if (Count == 0)
                 {
-                    throw new InvalidOperationException(
-                        "No RealmObjects. Has linker stripped them? See https://docs.mongodb.com/realm-legacy/docs/dotnet/latest/#linker-stripped-schema");
+                    var message = InteropConfig.Platform switch
+                    {
+                        InteropConfig.UnityPlatform => "Try weaving assemblies again ('Realm' -> 'Weave Assemblies' from the editor or simply make a code change) and make sure you don't have any Realm-related errors in the logs.",
+                        InteropConfig.DotNetPlatform => "Has linker stripped them? See https://docs.mongodb.com/realm-legacy/docs/dotnet/latest/#linker-stripped-schema",
```

Comment:
What is `them` referring to? Is there a message before this that always gets shown?

Reply:
Yes - on line 220 we throw `No RealmObjects. *this message*`

Reply:
Instead of looking into the past I should look into the future. How philosophical. :D 
I was only checking code before that part for some reason. Thanks for the hint! 🚀